### DISCUSSION TITLE
2024-03-21 kernel meeting notes

### DISCUSSION THREAD

**Discussion by zachschuermann** [delta-io/delta-kernel-rs#149] [General]
## summary
REPO IS NOW PUBLIC!!!!!! 🥳 

## action items
some PRs in flight:
- [ ] #146 get DAT working!
- [ ] #91 
- [ ] #129 (shouldn't be closed)
- [x] #109 
- [x] #131
- [x] #132
- [x] #130 
- [x] #141

### attendees
@nicklan @zachschuermann @hntd187 @ryan-johnson-databricks @tdas 

## notes stream
zach: hackathon update!
- did delta-sharing hackathon with nick and others! #144 
- delta-sharing branch: https://github.com/delta-io/delta-sharing/tree/2024q1-hackathon
- TLDR: delta-shared table with DVs can be read via delta-kernel-rs
- 1. delta-sharing changes to export local delta_log with presigned URLs
- 2. delta-kernel-rs python bindings so delta-sharing client can leverage rust kernel
- 3. delta-kernel-rs changes to read local delta log and presigned URLs
nick: #146
- discusson on: https://github.com/delta-incubator/delta-kernel-rs/pull/146/files#r1533878107
- add DAT case for microsecond/nanosecond read
